CHANGELOG.md

```diff
@@ -44,6 +44,10 @@ _None_
 * Strings: built-in templates now have better handling of multi-line translations.  
   [@mrackwitz](https://github.com/mrackwitz)
   [#774](https://github.com/SwiftGen/SwiftGen/pull/774)
+* Workaround for Mint that does not yet support the SPM resource bundle.  
```

Comment:
This is in the wrong place.

Reply:
Is the correct place for this an "Internal Changes"?

Reply:
No it's in the right "section", but your branch is behind on ours. 6.5 has been tagged and the changelog reset, put your changelog entry there.